src/Servant/CSV/Cassava/TSV.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE FlexibleInstances           #-}
+{-# LANGUAGE MultiParamTypeClasses       #-}
+{-# LANGUAGE DataKinds                   #-}
+{-# LANGUAGE KindSignatures              #-}
+{-# LANGUAGE OverloadedStrings           #-}
```

Comment:
Looks like you need `DeriveDataTypeable` for GHC 7.8 compatibility

Reply:
done.